MAiD - Case Management App/force-app/main/default/objects/Case/recordTypes/Error_Log.recordType-meta.xml

```diff
@@ -3,7 +3,7 @@
     <fullName>Error_Log</fullName>
     <active>true</active>
     <businessProcess>Error Log Support Process</businessProcess>
-    <label>Error Log</label>
+    <label>MAiD Error Log</label>
```

Comment:
This was updated to deal with MAiD concern that Error Log was the default, changing this label allowed MAiD Case to return to being the default in our testing. @RanadheerRG